src/player.cpp

```diff
@@ -3837,7 +3837,7 @@ void player::set_cat_level_rec(const std::string &sMut)
     if (!has_base_trait(sMut)) { //Skip base traits
         const auto &mdata = mutation_branch::get( sMut );
         for( auto &elem : mdata.category ) {
-            mutation_category_level[elem] += 8;
```

Comment:
Uhm, it increases the category level by 8 for each mutation, the number *is* important, search for where `mutation_category_level` is used. It is usually compared with fixed values like 40, 20 or so. 20 is reached with 3 mutations, after the changes here, it would require *20* mutations.